Tests.Integration.Presentation.Web/ItSystem/ItSystemUsageMigrationTests.cs

```diff
@@ -0,0 +1,45 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net;
+using System.Threading.Tasks;
+using Core.DomainModel.Organization;
+using Tests.Integration.Presentation.Web.Tools;
+using Xunit;
+
+namespace Tests.Integration.Presentation.Web.ItSystem
+{
+    public class ItSystemUsageMigrationTests
+    {
+
+        [Theory]
+        [InlineData(OrganizationRole.GlobalAdmin)]
+        [InlineData(OrganizationRole.User)]
+        public async Task Api_User_Can_Get_Public_Or_Own_Organization_It_Systems_Not_In_Use_In_Own_Organization(OrganizationRole role)
+        {
+            //Arrange
+            var itSystemName = CreateName();
+            var itSystem = await HttpApi.CreateItSystemInInitialOrganizationAsync(itSystemName);
+
+            var token = await HttpApi.GetTokenAsync(role);
+            var orgId = 1;
+            var url = TestEnvironment.CreateUrl($"api/ItSystemUsageMigration?organizationId={orgId}&nameContent={itSystemName[0]}&limit={10}");
```

Comment:
- din konstruktion er skrøbelig. Tag hele navnet med ellers risikerer du at testen fejler lige pludselig fordi der kommer flere itsystemer ind fra andre tests (du forventer kun 1 resultat og henter maks 10)

- Du mangler en test der opretter 2 IT systemer og har limit på 2 og validerer at der kommer 2 ud. Du opretter 2 for at sikre at der KAN komme 2 ud. Du kan så scope dem til at blive præcis dem du oprettede ved at give begge IT Systemer et unikt prefix dvs.
 PRefix = CreateName()
ITSystemname1 = Prefix + CreateNAme()
ITSystemname2 = Prefix + CreateNAme()
Når du så henter ud via dit endpoint giver du kun prefix med. Så bliver din test stabil og du får testet præcis det du gerne vil